charts/bcbox/values.yaml

```diff
@@ -113,6 +113,7 @@ config:
   configMap:
     FRONTEND_APIPATH: "api/v1"
     FRONTEND_COMS_APIPATH: ~
+    FRONTEND_NOTIFICATION_HOME_BANNER: ~
```

Comment:
not to be pedantic, but it might make sense to label this `FRONTEND_NOTIFICATION_BANNER`  and with property `.banner` in the env and components as it doesnt only appear on homepage (?)

Reply:
Yea after thinking about this some more, it might make more sense for the ENV variable name to just be `FRONTEND_NOTIFICATION_BANNER` as Tim recommended. In the json object, we'd want it to then just be `frontend.notificationBanner` or something similar. It'd be less verbose and would be able to cover more aspects and situations.

Reply:
Changing to
![image](https://user-images.githubusercontent.com/17445138/224391810-5d94233c-9f05-4574-aebc-11fc4d18570a.png)